src/main/java/io/lighty/yang/validator/checkupdatefrom/CheckUpdateFrom.java

```diff
@@ -780,17 +759,13 @@ private String buildTypeDefinitionPath(final TypeDefinition<?> typeDefinition,
             final SchemaInferenceStack schemaIS) {
         if (schemaIS.toInference().statementPath().size() == 1
                 && schemaIS.toInference().statementPath().get(0) instanceof TypedefEffectiveStatement) {
-            return schemaIS.toSchemaPath().toString();
+            return schemaIS.toString();
```

Comment:
Sure? What is a replacement for SchemaPath?

Reply:
Fixed, thanks